packages/generator-langium/templates/web/scripts/bundleClassic.js

```diff
@@ -0,0 +1,25 @@
+import { Uri } from 'vscode';
+import getConfigurationServiceOverride from '@codingame/monaco-vscode-configuration-service-override';
+import getEditorServiceOverride from '@codingame/monaco-vscode-editor-service-override';
+import getKeybindingsServiceOverride from '@codingame/monaco-vscode-keybindings-service-override';
+import { useOpenEditorStub } from 'monaco-languageclient';
+import { MonacoEditorLanguageClientWrapper } from 'monaco-editor-wrapper';
+import { addMonacoStyles } from 'monaco-editor-wrapper/styles';
```

Comment:
I'm not really sure why, but the types here is using [`css.d.ts`](https://github.com/TypeFox/monaco-components/blob/d7880556c4d69437a4ccd4ee071d48f4c21d32b0/packages/monaco-editor-wrapper/styles/css.d.ts) instead of [`index.d.ts`](https://github.com/TypeFox/monaco-components/blob/d7880556c4d69437a4ccd4ee071d48f4c21d32b0/packages/monaco-editor-wrapper/styles/index.d.ts):

![image](https://github.com/eclipse-langium/langium/assets/87622592/c5fa3b09-3380-4746-a2e6-0e448dfd2e77)

After using the editor to see types, it opens this file:

![image](https://github.com/eclipse-langium/langium/assets/87622592/17c375b0-3b91-490f-b75a-de1ff75e4178)

Reply:
`typesVersions` were incorrectly specified. Fixed in latest commit/force-push.